src/platform_impl/windows/window_state.rs

```diff
@@ -116,6 +116,8 @@ bitflags! {
         /// Drop shadow for undecorated windows.
         const MARKER_UNDECORATED_SHADOW = 1 << 20;
 
+        const MARKER_NO_ACTIVATE = 1 << 18;
```

Comment:
Should invert the flag as it's easier to read imo and adjust the logic accordingly - `MARKER_ACTIVATE`